spec/liquid_tags/glitch_tag_spec.rb

```diff
@@ -75,6 +75,13 @@ def generate_tag(id)
       expected = URI.parse("https://glitch.com/embed/#!/embed/some-id?path=index.html")
       expect(actual).to eq(expected)
     end
+
+    it "generates a glitch URI" do
+      template = generate_tag(id_with_app_option)
+      string = template.root.nodelist.first.uri
```

Comment:
This doesn't really fix the regression, because as you can see in the other tests `template.root.nodelist.first.uri` is a Ruby object containing the URI.

What you want to test in my opinion is that calling render on the template injects the URI inside the HTML.

Something like:

`expect(template.render(nil)).to include(URI)`

this way you check the actual HTML string

Reply:
with that approach, do you think it would be better to not make a specific regression test for valid uris, and instead always check that the output contains `src="#{expected}"`?